packages/donor/src/screens/UpcommingDonation/EventCalendarLink.tsx

```diff
@@ -62,17 +61,42 @@ export default function EventCalendarLink({
     window.open(googleEventUrlWithParams, "_blank");
   };
 
+  const addToIOSCalendar = () => {
+    const calendarUrl = [
+      "BEGIN:VCALENDAR",
+      "VERSION:2.0",
+      "BEGIN:VEVENT",
+      "URL:" + document.URL,
+      "DTSTART:" + CleanIsoFormatDate(eventDateStart),
+      "DTEND:" + CleanIsoFormatDate(eventDateEnd),
+      "SUMMARY:" + "תרומת טרומבוציטים",
+      "DESCRIPTION:" + eventDescription,
+      "LOCATION:" + eventLocation,
+      "END:VEVENT",
+      "END:VCALENDAR",
+    ].join("\n");
+
+    const filename = "download.ics";
+    const blob = new Blob([calendarUrl], {
+      type: "text/calendar;charset=utf-8",
+    });
+    const link = document.createElement("a");
+    link.href = window.URL.createObjectURL(blob);
+    link.setAttribute("download", filename);
+    document.body.appendChild(link);
+    link.click();
+    document.body.removeChild(link);
```

Comment:
Why do you need to create an `a` element here, and not in line 99?

Would this work?

```
        <div className={styles.link} >
               <a href={...} download={...}>הוספה ליומן</a>
        </div>
```

Reply:
fixed